src/Illuminate/Foundation/Bootstrap/HandleExceptions.php

```diff
@@ -38,7 +38,9 @@ public function bootstrap(Application $app)
 
         $this->app = $app;
 
-        error_reporting(-1);
+        if (! $app->environment('production')) {
+            error_reporting(-1);
+        }
```

Comment:
I honestly thought about removing this line but it exists since forever - for a reason which I couldn't find in the history.